scripts/build/debian/config.json

```diff
@@ -4,7 +4,7 @@
   "genericName": "Etcher",
   "homepage": "https://etcher.io/",
   "icon": "assets/icon.png",
-  "bin": "etcher-electron.sh",
+  "bin": "etcher",
```

Comment:
Actually, I'm reverting the changes. This file has a `name` property at the top which is set to `etcher-electron`, so it doesn't really matter what the name of the binary entry point it, as it will always be aliases to the `name`.